nupic/research/frameworks/pytorch/imagenet/imagenet_tune.py

```diff
@@ -442,20 +484,28 @@ def run_single_instance(config):
     config["reuse_actors"] = False
     config["dist_port"] = get_free_port()
 
+    # set trainable
+    ray_trainable = config.get("ray_trainable", SupervisedTrainable)
+    # assert issubclass(ray_trainable, BaseTrainable)
```

Comment:
Why did you remove this assert?

Reply:
I will reintroduce as `assert issubclass(ray_trainable, Trainable)`. We now can have trainables that are not subclassed from `BaseTrainable` but from `Trainable` directly.